doc/Tutorials/Dynamic_Map.ipynb

```diff
@@ -15,25 +15,33 @@
     "\n",
     "The ``DynamicMap`` addresses these issues by computing and displaying elements dynamically, allowing exploration of much larger datasets:\n",
     "\n",
-    "* DynamicMaps generate elements on the fly allowing the process of exploration to begin immediately.\n",
+    "* DynamicMaps generate elements on the fly, allowing the process of exploration to begin immediately.\n",
     "* DynamicMaps do not require fixed sampling, allowing exploration of parameters with arbitrary resolution.\n",
     "* DynamicMaps are lazy in the sense they only compute only as much data as the user wishes to explore.\n",
     "\n",
     "The limitations of ``DynamicMaps`` are:\n",
     "\n",
-    "* DynamicMaps require a live notebook server and cannot be directly exported to static HTML.\n",
-    "* DynamicMaps do not hold data which reduces the utility of a pickling DynamicMap.\n",
-    "* DynamicMaps are often stateful as are their element caches.\n",
+    "* DynamicMaps require a live notebook server and cannot be fully exported to static HTML.\n",
+    "* DynamicMaps do not store copies of the underlying data, which reduces the utility of pickling a DynamicMap.\n",
```

Comment:
The point this is making holds but technically it does hold copies of the data in the cache.